### PR TITLE
Update power on blade detail view

### PR DESCRIPTION
**Pull request by juliasaveliff**

### CODE REVIEW

File: src/components/elementView/powerView/powerView.tsx [hunk 1]
```diff
@@ -141,6 +142,26 @@ export class PowerView extends React.PureComponent<
     this.props.updated();
   }
 
+  private isAssetPowerNetworkControlled() {
```
Comment (juliasaveliff) [inline src/components/elementView/powerView/powerView.tsx:145]: wrote this helper method to hopefully help fix the bug but not calling it anywhere

File: rackcity/views/power_views.py [hunk 2]
```diff
@@ -552,7 +552,8 @@ def make_bcman_request(chassis, blade, power_command):
     result = None
     if os.path.exists("temp.txt"):
         fp = open("temp.txt", "r")
-        result = fp.read().splitlines()[0]
+        result = " ; ".join(fp.read().splitlines())
+        # result = fp.read().splitlines()[0]
```
Comment (juliasaveliff) [inline rackcity/views/power_views.py:556]: these endpoints are giving me trouble so i'm going to keep debugging and push fixes in another PR

**Comment by clairecfu** [comment]
can u fix the ci build though? probably linting
